apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:200,400,900');
+
+*{
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-size: 24px;
+  color:#4d4d4d;
+}
+
+.hero-header{
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background:url(../images/apple-pie.jpg);
+  background-size: cover;
+}
+#hero-header-french{
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background:url(../../french-onion-soup-starter-code/images/soup.jpg);
+  background-size: cover;
+}
+
+.hero-header header{
+  position:relative;
+  top: 50%;
+  transform:translateY(-50%);
+}
+.hero-header header h1{
+  font-size: 7vw;

```

Comment:
Responsive typography is a hard thing, try to use fixed units for typography and not relative units.